kernel/Dockerfile

```diff
@@ -61,8 +65,17 @@ RUN curl -fsSLO ${KERNEL_SHA256_SUMS} && \
     gpg2 --verify linux-${KERNEL_VERSION}.tar.sign linux-${KERNEL_VERSION}.tar && \
     cat linux-${KERNEL_VERSION}.tar | tar --absolute-names -x && mv /linux-${KERNEL_VERSION} /linux
 
-# Apply local patches if present
 WORKDIR /linux
+# Apply Preempt-RT kernel patches in the case of RT Linux Kernel
+RUN set -e && \
+    if [ -d /patches-${KERNEL_SERIES}${EXTRA} ]; then \
+	echo "Building ${EXTRA} kernel"; \
+	for line in `sed -e "s/#.*//g" /patches-${KERNEL_SERIES}${EXTRA}/series`; do \
+	    patch -p1 < /patches-${KERNEL_SERIES}${EXTRA}/$line; \
+	done; \
+    fi
+
+# Apply local patches if present
```

Comment:
This comment is no longer correct. Also, wouldn't it make more sense to apply the common patches first and then apply the extra patches on top?

Reply:
# Apply local common patches if present  Does this work for you?